engine/gameobject/src/gameobject/comp_anim.cpp

```diff
@@ -390,7 +390,7 @@ namespace dmGameObject
 
     static AnimWorld* GetWorld(HCollection hcollection)
     {
-        dmResource::ResourceType resource_type;
+        HResourceType resource_type;
```

Comment:
Alternative could have been the new typedef `dmResource::HType`, but I kind of want to use the C api to a good extent internally as well.